anomalib/models/padim/torch_model.py

```diff
@@ -42,7 +42,7 @@ def __init__(
 
         self.backbone = backbone
         self.layers = layers
-        self.feature_extractor = FeatureExtractor(backbone=self.backbone, layers=layers, pre_trained=pre_trained)
+        self.feature_extractor = TimmFeatureExtractor(backbone=self.backbone, layers=layers, pre_trained=pre_trained)
```

Comment:
Why don't we switch to the torch fx feature extractor in our models?

Reply:
I don't mind doing that.

Reply:
Let's keep it for now till otx stuff is sorted.